lib/rules/max-line-length/__tests__/index.js

```diff
@@ -238,7 +238,7 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
-  config: [ 20, { ignorePattern: "/https?:\/\/[0-9,a-z]*.*/" } ],
+  config: [ 20, { ignorePattern: "/https?://[0-9,a-z]*.*/" } ],
```

Comment:
Looks like the string has the beginning and ending `/` cut off of it and fed into `new RegExp` [here](https://github.com/stylelint/stylelint/blob/8abafb8fe621ada73552cd9fe436553df5b94b53/lib/utils/matchesStringOrRegExp.js#L48), which means the `/` within the string do _not_ need to be escaped:
![image](https://user-images.githubusercontent.com/427137/28177791-d6dbf358-67b0-11e7-8e48-0807487d0b8a.png)

So this change is correct as far as I can see 😉.